### DISCUSSION TITLE
Is it really multi-vendor system?

### DISCUSSION THREAD

**Discussion by bdsumon4u** [shopperlabs/framework#185] [Q&A] [answered]
It claims that it is 

> An eCommerce administration built with Laravel to create and manage online shop with **multi-vendor**.

But I can't find the vendor system.

**Comment by mckenziearts** [comment, marked as answer]
Hey @bdsumon4u, closing the loop on this one. Multi-vendor is on the roadmap and in active development, no official release date yet. The older README framing that mentioned "multi-vendor" predated the current architecture and has since been reworded. Consolidating the conversation over at #352 for updates.
